apis/field_error.go

```diff
@@ -251,6 +256,16 @@ func (fe *FieldError) normalized() []*FieldError {
 	return errors
 }
 
+// WrapErrors returns the value of the nested errors.
+func (fe *FieldError) WrapErrors() []FieldError {
```

Comment:
I think the Hashicorp library called with `WrappedErrors` not `WrapErrors`, which reads differently.

Reply:
Done. I meant to rename to `WrappedErrors`, thanks for catching.